osu.Game.Rulesets.Catch/Resources/Testing/Beatmaps/basic-expected-conversion.json

```diff
@@ -1270,6 +1270,111 @@
                     "Position": 408.720825
                 }
             ]
-        }
+        },
+        {
+            "StartTime": 25000,
+            "Objects": [
+                {
+                    "StartTime": 25000,
+                    "Position": 448
```

Comment:
This is a question but why this `"Position"` field is an integer while an another field above has decimals?

Reply:
This is because this specific slider is completely vertical (all points have X axis of 448), and because droplets always get shifted by rand(-20, 20) integer, they will all actually be integers (stored as a float, of course).